server/views/partials/layout.njk

```diff
@@ -16,6 +17,21 @@
     <!-- End Google Analytics -->
   {% endif %}
 
+  {# Google Analytics: Google Analytics 4 (GA4) #}
+  {% if googleAnalyticsGa4Id %}
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsGa4Id }}"></script>
+    <script nonce="{{ cspNonce }}">
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', '{{ googleAnalyticsGa4Id }}', {
+        active_case_load: '{{ user.activeCaseload.id | default('') }}',
```

Comment:
This property will be automagically added to all events by GA4, similarly to what done previously in UA via a custom dimension.